examples/embeddable_examples/public/react_embeddables/saved_book/create_saved_book_action.tsx

```diff
@@ -42,14 +41,14 @@ export const registerCreateSavedBookAction = (uiActions: UiActionsPublicStart, c
         core,
       });
 
-      const bookAttributes = serializeBookAttributes(newPanelStateManager);
+      const bookAttributes = newPanelStateManager.getLatestState();
       const initialState: BookRuntimeState = savedBookId
```

Comment:
initialState should be typed as BookSerializeState, `BookRuntimeState` is no longer exposed to parent.